ClassNotFoundException for LRUMap (commons collections) while running the interpreter

This happens:

```
0    [main] ERROR roject.wollok.launch.WollokLauncher  - Lorg/apache/commons/collections/map/LRUMap;
java.lang.NoClassDefFoundError: Lorg/apache/commons/collections/map/LRUMap;
	at java.lang.Class.getDeclaredFields0(Native Method)
	at java.lang.Class.privateGetDeclaredFields(Class.java:2570)
	at java.lang.Class.getDeclaredFields(Class.java:1903)
	at com.google.inject.spi.InjectionPoint.getInjectionPoints(InjectionPoint.java:649)
	at com.google.inject.spi.InjectionPoint.forInstanceMethodsAndFields(InjectionPoint.java:356)
	at com.google.inject.internal.ConstructorBindingImpl.getInternalDependencies(ConstructorBindingImpl.java:151)
	at com.google.inject.internal.InjectorImpl.getInternalDependencies(InjectorImpl.java:585)
	at com.google.inject.internal.InjectorImpl.cleanup(InjectorImpl.java:543)
	at com.google.inject.internal.InjectorImpl.initializeJitBinding(InjectorImpl.java:529)
	at com.google.inject.internal.InjectorImpl.createJustInTimeBinding(InjectorImpl.java:847)
	at com.google.inject.internal.InjectorImpl.createJustInTimeBindingRecursive(InjectorImpl.java:772)
	at com.google.inject.internal.InjectorImpl.getJustInTimeBinding(InjectorImpl.java:256)
	at com.google.inject.internal.InjectorImpl.getBindingOrThrow(InjectorImpl.java:205)
	at com.google.inject.internal.InjectorImpl.getInternalFactory(InjectorImpl.java:853)
	at com.google.inject.internal.FactoryProxy.notify(FactoryProxy.java:46)
	at com.google.inject.internal.ProcessedBindingData.runCreationListeners(ProcessedBindingData.java:50)
	at com.google.inject.internal.InternalInjectorCreator.initializeStatically(InternalInjectorCreator.java:133)
	at com.google.inject.internal.InternalInjectorCreator.build(InternalInjectorCreator.java:106)
	at com.google.inject.Guice.createInjector(Guice.java:95)
	at com.google.inject.Guice.createInjector(Guice.java:72)
	at com.google.inject.Guice.createInjector(Guice.java:62)
	at org.uqbar.project.wollok.launch.setup.WollokLauncherSetup.createInjector(WollokLauncherSetup.java:36)
	at org.uqbar.project.wollok.WollokDslStandaloneSetupGenerated.createInjectorAndDoEMFRegistration(WollokDslStandaloneSetupGenerated.java:23)
	at org.uqbar.project.wollok.launch.WollokChecker.doMain(WollokChecker.java:76)
	at org.uqbar.project.wollok.launch.WollokLauncher.main(WollokLauncher.java:31)
Caused by: java.lang.ClassNotFoundException: org.apache.commons.collections.map.LRUMap
	at java.net.URLClassLoader$1.run(URLClassLoader.java:372)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:361)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:360)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:308)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 25 more
```
when trying tu run a program or REPL.

It is not resolving the commons.collection dependency

--- a/org.uqbar.project.wollok/META-INF/MANIFEST.MF
+++ b/org.uqbar.project.wollok/META-INF/MANIFEST.MF
@@ -28,7 +28,7 @@ Require-Bundle: org.eclipse.xtext;bundle-version="2.9.0";visibility:=reexport,
  org.eclipse.jdt,
  org.eclipse.jdt.core,
  org.uqbar.project.xinterpreter,
- org.apache.commons.collections;bundle-version="3.2.2"
+ org.apache.commons.collections;bundle-version="3.2.2";visibility:=reexport
 Import-Package: org.eclipse.xtext.xbase.lib,
  org.apache.log4j
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8

